tests/openssl_sgx_tcrypto_compat/host/CSchema_checker.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "include_openssl.h"
+#include "string.h"
+
+#define O_ENCLAVE 0
+#define O_NATIVE 1
```

Comment:
Where are these defines used?  (I couldn't find anywhere)  Remove if unused.

Reply:
There used to be testing for both trusted and untrusted space. The definitions are no more used in the code now. I can remove them.